src/plugins/dashboard/public/dashboard_container/embeddable/create/create_dashboard.ts

```diff
@@ -635,5 +636,11 @@ export const initializeDashboard = async ({
     });
   }
 
+  if (loadDashboardReturn.dashboardId && !incomingEmbeddable) {
+    // TODO: improve this?
+    // For example, when a user is editing a dashboard an is returning from an editor by canceling, we shouldn't count this as a new view
+    dashboardContentInsights.trackDashboardView(loadDashboardReturn.dashboardId);
```

Comment:
Hmm... I wonder, should we count a view only in `view` mode for a dashboard? Should "views" in edit mode count? cc @teresaalvarezsoler This feels like a product question

Reply:
Agree edit mode should also count. We should count as `view` when users open the dashboard regardless of the edit/view modes. We know for sure that many users use the dashboard only in edit mode, we would be massively underestimating usage if we don't consider these.

Btw, we defined what should be count as `view` here https://github.com/elastic/kibana/issues/183687#issuecomment-2220291067

Reply:
I just tested the following scenarios and it works very well:
- It counts as view every time the dashboard is open (in the same or another tab) regardless of the edit/view modes
- It doesn't count as view when users creates maps or lens charts from the dashboard and come back to the dashboard
- It count as view any user opening the dashboard regardless of their editor/viewer roles
Good job!

Reply:
I looked into this offline, and it would be possible to **not** count cancelling if we modified the embeddable state transfer service to include some cancelling logic - however, this is pretty large addition and, after talking with @teresaalvarezsoler and @ThomThomson, we agreed that this is indeed not worth the additional effort at the point.

@Dosant Would it be okay if you could create a follow-up issue and link it in the `TODO` comment here? We have plans to move away from the embeddable transfer service in general, so it would be good to have an issue that we can keep track of

Reply:
@Heenawter, thank you! I'll clarify the comment and will create an issue

Reply:
https://github.com/elastic/kibana/issues/190485